test/e2e_node/resource_metrics_test.go

```diff
@@ -111,9 +121,15 @@ var _ = framework.KubeDescribe("ResourceMetricsAPI", func() {
 })
 
 func getV1alpha1ResourceMetrics() (e2emetrics.KubeletMetrics, error) {
```

Comment:
Just rename function name to `getResourceMetrics` will be fine.

Reply:
yea, that's an earlier version